app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+    for i in phrase:
+        if i.lower() == letter.lower():
+            count += 1
+
+    return count
```

Comment:
Try to write it in one line
use count()

Reply:
fixed